service/plugin/plugin.go

```diff
@@ -58,7 +59,7 @@ import (
 
 // Config represents configuration for the soci snapshotter plugin.
 type Config struct {
-	service.Config
+	config.ServiceConfig
```

Comment:
what's the difference between this two config? (`plugin.Config` v.s. `config.ServiceConfig`) (e.g., can we move this to the `config` pkg as well?)

Reply:
I'm not sure why we need `plugin.go`. It looks like its purpose to register the snapshotter with containerd and CRI. Although that is already done in `main.go` within the snapshotter - https://github.com/awslabs/soci-snapshotter/blob/main/cmd/soci-snapshotter-grpc/main.go#L230. Also the package itself is never imported. https://github.com/search?q=repo%3Aawslabs%2Fsoci-snapshotter+github.com%2Fawslabs%2Fsoci-snapshotter%2Fservice%2Fplugin&type=code